### DISCUSSION TITLE
[tsml_react] Shortcode Not Working...

### DISCUSSION THREAD

**Discussion by teach4learning** [code4recovery/12-step-meeting-list#1152] [Q&A] [answered]
Using 3.14.16 plugin @ https://cbiaa.org.

“Meetings in Progress” not appearing / showing, nor do I see place to trigger in settings.

What am I missing?

gb / intergroup web person

**Comment by kiyote33** [comment]
Hi @teach4learning, I'm not sure what you asking so I took a look at your site. I see you are using TSML_UI on your meetings page [https://www.cbiaa.org/meetings/]. The "Meetings in Progress" feature appears to be working as expected, so either the glitch you were experiencing went away or maybe there is some other issue that you need help with. If the latter is the case, please expand on it and we'll try to assist you.

**Comment by teach4learning** [comment]
Thank you. I had been using [tsml_react] as shortcode, then changed it as I
attempted to solve problem. didn't wait long enough to recheck; and now
it's working.

I appreciate your prompt feedback.

fyi in all that I could find about set up and new install I did not see any
instructions to new users about shortcode to use on meeting page. I"m sure
there was some guidance back when I first installed years ago, but find
none now. And memory is no longer one of my stonger suits!

Thanks again,

gb

**Reply by kiyote33** [reply, marked as answer]
Ah! For posterity & other new users: I believe both [tsml_react] and [tsml_ui] will work as shortcodes to access our latest front end (that we call **TSML UI**) on your custom HTML page. You can also use the **Meetings/Settings/User Interface Display** feature to switch back and forth between it and our standard **Legacy UI** on the default /meetings page. Glad it's working for you now!
